src/main/java/com/limechain/lightclient/FullClient.java

```diff
@@ -0,0 +1,41 @@
+package com.limechain.lightclient;
```

Comment:
What about renaming the package from `lightclient` to something more fitting, like just `client`, or `node` or something... I'm not sure?

Reply:
@vikinatora

Reply:
Makes sense 👍

Reply:
We'll go with `client`.